Bottom: 6f22d8801eb0c12043db0aa33a7b3c58a73fcd12
Top:    8a663ad426ea58df81a88381b20c2edeed2cac7c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-12 22:21:01 +0100

Refresh of try-something-with-the-rtrees

---

diff --git a/src/find.c b/src/find.c
index fb06e23..4e8d4f0 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2536,7 +2536,6 @@ LookupLOConnectionsToPolygon (PolygonTypePtr Polygon, Cardinal LayerGroup)
             }
         }
     }
-
   return (False);
 }
